<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MateriaBasicaWindow</class>
 <widget class="QMainWindow" name="MateriaBasicaWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Materia Básica</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_horario_basica">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Horario:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_horario_basica">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_periodo_basica">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Periodo:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_periodo_basica">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_creditos_basica">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Créditos:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_creditos_basica">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_htrabajo_basica">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Horas trabajo:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_htrabajo_basica">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_hlaboratorio_basica">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Horas laboratorio:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_hlaboratorio_basica">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_nombre_basica">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nombre:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_nombre_basica">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_area_basica">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Área:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_area_basica">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_hteoria_basica">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Horas teoria:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_hteoria_basica">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_cursada_basica">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Es cursada por:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_cursada_basica">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>270</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_ensenada_basica">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Es enseñada por:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_ensenada_basica">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_requiere_basica">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Requiere:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_requiere_basica">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>330</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_agregar_basica">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_eliminar_basica">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>410</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_mensaje_basica">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>360</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Mensaje</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>508</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
